src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/creategeneralorder/CreateGeneralOrderAboutToStartHandler.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler.creategeneralorder;
+
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackHandler;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.GeneralOrderWrapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.IdamService;
+
+import java.time.Clock;
+import java.time.LocalDate;
+
+abstract class CreateGeneralOrderAboutToStartHandler extends FinremCallbackHandler {
+
+    private final IdamService idamService;
+    private final Clock clock;
+
+    public CreateGeneralOrderAboutToStartHandler(FinremCaseDetailsMapper finremCaseDetailsMapper,
```

Comment:
i think protected constructor would be better

Reply:
Just tried this but it's causing the EventTypeTest to fail so changing back to public